Rakefile

```diff
@@ -1,39 +1,100 @@
-require "rake/clean"
-require "bundler/gem_tasks"
-
-CLOBBER.include "pkg", "node_modules"
-
-# add npm-install as pre-req for build
-Rake::Task[:build].enhance [:npm]
-
-###########
-# npm
-###########
-desc "alias for npm:install"
-task :npm => "npm:install"
-namespace :npm do
-  desc "Install npm dependencies"
-  task :install do
-    sh "npm install"
+# bootstrap
+desc "bootstrap all packages"
+task :bootstrap, [:pkg] do |t, args|
+  if args[:pkg].nil?
+    sh "
+    baseDir=$PWD
+    for dir in ./packages/*
+    do
+      cd $dir
+      bundle install
+      cd $baseDir
+    done
+    "
+  else
+    pkgDir = "./packages/#{args[:pkg]}"
+    sh "cd #{pkgDir} && bundle install"
   end
+end
 
-  desc "Update npm dependencies to latest version allowed by package.json"
-  task :update do
-    sh "npm update"
+# test
+desc "Test all packages"
+task :test, [:pkg] do |t, args|
+  if args[:pkg].nil?
+    sh "
+    baseDir=$PWD
+    for dir in ./packages/*
+    do
+      cd $dir
+      rake test
+      cd $baseDir
+    done
+    "
+  else
+    pkgDir = "./packages/#{args[:pkg]}"
+    sh "cd #{pkgDir} && rake test"
   end
+end
 
-  desc "Upgrade axe-core dependency to latest version available, overwriting package.json"
-  task :upgrade do
-    sh "npm install --save axe-core@latest"
+# build
+desc "Build all packages"
+task :build, [:pkg] do |t, args|
+  if args[:pkg].nil?
+    sh "
+    baseDir=$PWD
+    for dir in ./packages/*
+    do
+      cd $dir
+      rake build
+      cd $baseDir
+    done
+    "
+  else
+    pkgDir = "./packages/#{args[:pkg]}"
+    sh "cd #{pkgDir} && rake build"
   end
+end
 
-  desc "Upgrade axe-core dependency to latest prerelease version, overwriting package.json"
-  task :next do
-    sh "npm install --save axe-core@next"
+# format
+desc "format code in all packages"
+task :format, [:pkg] do |t, args|
+  if args[:pkg].nil?
+    sh "
+    baseDir=$PWD
+    for dir in ./packages/*
+    do
+      cd $dir
+      rake format
+      cd $baseDir
+    done
+    "
+  else
+    pkgDir = "./packages/#{args[:pkg]}"
+    sh "cd #{pkgDir} && rake format"
   end
+end
 
-  desc "Display currently-installed and latest-available versions of axe-core lib"
-  task :status do
-    sh "npm view axe-core version && npm list axe-core"
+# clobber
+desc "clobber/clean all packages"
+task :clobber, [:pkg] do |t, args|
+  if args[:pkg].nil?
+    sh "
+    baseDir=$PWD
+    for dir in ./packages/*
+    do
+      cd $dir
+      rake clobber
+      cd $baseDir
+    done
+    "
+  else
+    pkgDir = "./packages/#{args[:pkg]}"
+    sh "cd #{pkgDir} && rake clobber"
   end
 end
+
+# format code
+desc "Format code using rubocop"
+task :format => [] do
+  sh "bundle exec rubocop --config rubocop.yml --auto-correct --fix-layout"
```

Comment:
It looks like `rubocop.yml` was deleted. Is that intended?

Reply:
I see packages contain a `rubocop.yml` so I'm assuming that's the config that is supposed to be used

Reply:
You are right. Copy pasta error this one. 
Individual packages have their own rubocop, and there is a helper task for that from here - https://github.com/dequelabs/axe-core-gems/pull/120/files#diff-52c976fc38ed2b4e3b1192f8a8e24cffR60